src/sentry_plugins/opsgenie/plugin.py

```diff
@@ -35,7 +36,7 @@ class OpsGenieOptionsForm(notify.NotificationConfigurationForm):
     )
 
 
-class OpsGeniePlugin(notify.NotificationPlugin):
+class OpsGeniePlugin(CorePluginMixin, notify.NotificationPlugin):
```

Comment:
Extending `CorePluginMixin` allows us to `raise_error` which does some useful mappings on errors.